src/libraries/System.Security.Cryptography.X509Certificates/tests/CollectionTests.cs

```diff
@@ -1426,6 +1427,55 @@ public static void SerializedCertDisposeDoesNotRemoveKeyFile()
             }
         }
 
+        [Fact]
+        public static void ImportFromPem_SingleCertificate_Success()
+        {
+            X509Certificate2Collection cc = new X509Certificate2Collection();
+            cc.ImportFromPem(PemTestData.ECDsaCertificate);
+            Assert.Single(cc);
+            Assert.Equal("E844FA74BC8DCE46EF4F8605EA00008F161AB56F", cc[0].Thumbprint);
+        }
```

Comment:
Please ensure that all the certificates loaded in these tests are disposed.

Probably new methods on the "Cert" helper type to return the properly managed ImportedCollection.